src/ray/gcs/accessor.h

```diff
@@ -106,11 +106,12 @@ class ActorInfoAccessor {
 
   /// Get actor checkpoint data from GCS asynchronously.
   ///
+  /// \param actor_id The ID of actor to look up in the GCS.
   /// \param checkpoint_id The ID of checkpoint to lookup in GCS.
   /// \param callback The callback that will be called after lookup finishes.
   /// \return Status
   virtual Status AsyncGetCheckpoint(
-      const ActorCheckpointID &checkpoint_id,
+      const ActorID &actor_id, const ActorCheckpointID &checkpoint_id,
```

Comment:
maybe put `actor_id` as the second argument?

Reply:
fixed